### PR TITLE
#1079 Added description of src attribute to py-script

### PR DESCRIPTION
**Pull request by Kd-Here**
Hey @JeffersGlass added description for `src` attribute as per your suggestion in #1079

### CODE REVIEW

**Comment by Kd-Here** [comment]
@JeffersGlass 
@FabioRosado Could you review the PR
**I added `src` description as per Jeff guidance**

File: docs/reference/elements/py-script.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The `<py-script>` element lets you execute multi-line Python scripts both inline
 
 | attribute | type   | default | description                  |
 |-----------|--------|---------|------------------------------|
-| **src**   | url    |         | Url of a python source file. |
+| **src**   | url    |         | You don't need to add long python code in py-script, you can provide url of the python source file in py-script tag with `src` attribute. When a Python file is referred with `src` attribute it is executed, and then added to the namespace where it was referred. |
```
Comment (FabioRosado) [inline docs/reference/elements/py-script.md:9]:
```suggestion
| **src**   | url    |         | You don't need to add long python code in py-script, you can provide url of the python source file in the py-script tag with the `src` attribute. When a Python file is referred with the `src` attribute it is executed, and then added to the namespace where it was referred. |
```

**Review by FabioRosado — APPROVED** [review]
Hello @Kd-Here  apologies for the very long delay in getting back to you.

I think this slipped through the cracks, I've added some suggestions, I'll comment them and merge the PR.

Thank you so much for working on this and opening up the PR!

**Comment by Kd-Here** [comment]
Thansk @FabioRosado for reviewing it.
I really appreciate your overall efforts.
@JeffersGlass you too.

**Pull request merged**